src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure implements AreaCalculator, FigureDraw {
```

Comment:
```suggestion
public class Rectangle extends Figure {
```
в дочірньому класі не потрібно повторювати implements AreaCalculator, FigureDraw так як ці типи переходять від Figure автоматично